test/mysql.test.js

```diff
@@ -0,0 +1,34 @@
+const expect = require('expect.js');
+
+const mysqlComponent = require('..');
+
+describe('Mysql Component', () => {
+  let pool;
+  let connection;
+
+  before( async () => {
+    let dependencies = {
+      config: {
+        connectionLimit: 10,
+        host: 'localhost',
+        user: 'admin',
+        password: 'password',
+        database: 'event_aggregates'
+      }
+    };
+    mysql = mysqlComponent(dependencies);
+    pool = await mysql.start(dependencies.config);
+    mysql.createConnec
```

Comment:
Is this right? Maybe something incomplete?

Reply:
fixed